docs/branching-and-tagging.md

```diff
@@ -101,13 +101,13 @@ ALTER TABLE db.table CREATE BRANCH `audit-branch` AS OF VERSION 3 RETAIN 7 DAYS
 3. Writes are performed on a separate `audit-branch` independent from the main table history.
 ```sql
 -- WAP Branch write
-SET spark.wap.branch = 'audit-branch'
+SET spark.wap.branch = audit-branch
```

Comment:
note in the previous example branch `'audit-branch'` is created instead of `audit-branch`.